app.py

```diff
@@ -43,6 +38,36 @@ def group_category(group=[], category='all'):
         result["count"] = len(posts)
         result["query"] = search
         return flask.render_template('search.html', result=result)
+
+    page = flask.request.args.get('page')
+    posts, metadata = api.get_posts(page=page, per_page=13)
+
+    webinars = get_rss_feed_content(
+        'https://www.brighttalk.com/channel/6793/feed/rss'
+    )
+
+    featured_post = api.get_featured_post()
+    homepage_posts = []
+
+    for post in posts:
+        if post['id'] != featured_post['id']:
```

Comment:
could there ever be more than one featured_post? I think we only need to remove the first one

Reply:
This is looking through the posts and checking it does not match the featured post.

Reply:
is there only one of them ever?

Reply:
`api.get_featured_post()` only returns one post by default. Although there are many featured posts the featured post is the most recent one.